requirements.txt

```diff
@@ -1,6 +1,6 @@
 coverage < 4.0.0
-django >= 1.4.11, < 1.7
-djangorestframework > 3.0.0, < 3.3.0
+Django >= 1.10
```

Comment:
Could not import it to Yolapi for some reason, and didn't want this to block me.